### ISSUE TITLE
Allow fields to be planted in rows

### ISSUE THREAD

**Issue by dhouck** [ldtteam/minecolonies-features#420]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version: 1.16.5
- MineColonies Version: 0.14.156 Alpha
- Related Mods and their Versions: Structurize 0.13.171 Alpha

## Is your feature request related to a problem?

Farmers and field definitions are only half the speed they could be.  Crops grow twice as fast when planted in rows (technical definition: none of the same kind of crop is diagonal, and if any is to the north or south it isn't to the east or west and vice versa).  Currently, the only way to do that in MineColonies is to define several 1x11 fields next to each other, which requires more farmers than if you had them as a single field.

## Enhancement or Change description

A single field should be able to contain multiple crops and have the farmer automatically plant those crops in rows.

### Notes or related things

 * Ideally the direction and arrangement of the rows would be controllable from the farm GUI
 * It technically isn't exactly twice as fast because growth rate is more complicated, but it should be about that; the full details are [on the wiki](https://minecraft.fandom.com/wiki/Tutorials/Crop_farming#Growth_rate "Minecraft Wiki crop farming page")

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by ravenbuilder934** [comment]
> Farmers and field definitions are only half the speed they could be.

this is intentional. Citizens aren't machines, they aren't going to be as efficient as a player or tech mod. Their strength is that they can keep going when the player isn't there

**Comment by dhouck** [comment]
That's fair and I probably shouldn't have phrased it that way; I'm not talking about the speeds that can be increased with Speed or Athletics.

But citizens aren't machines; they're supposed to be more like people.  And people can learn to do things better.  It seems odd that a skilled, experienced farmer, who knows how to make 160 farm-related recipes, isn't able to learn to plant crops in ways that make them grow better.

**Comment by ravenbuilder934** [comment]
perhaps at higher levels they can do this?

**Comment by dhouck** [comment]
Yeah, that was my thought.  As farms get higher level they can handle multiple crops per field, not just more fields, or something.
